big_tests/tests/service_mongoose_system_metrics_SUITE.erl

```diff
@@ -245,6 +281,27 @@ system_metrics_service_is_enabled(Node) ->
 system_metrics_service_is_disabled(Node) ->
     not system_metrics_service_is_enabled(Node).
 
+configure_additional_tracking_id(Node) ->
+    TrackingIdArgs = [google_analytics_tracking_id, ?TRACKING_ID_ADDITIONAL],
+    {atomic, ok} = mongoose_helper:successful_rpc(Node, ejabberd_config, add_local_option, TrackingIdArgs).
+
+remove_additional_tracking_id(Node) ->
+    mongoose_helper:successful_rpc(Node, ejabberd_config, del_local_option, [ google_analytics_tracking_id ]).
+
+events_are_reported_to_additional_tracking_id() ->
+    Tab = ets:tab2list(?ETS_TABLE),
+    UniqueSortedTab = lists:usort([Tid ||#event{tid = Tid} <- Tab]),
```

Comment:
you do not need to sort, you can just make a set from this list: http://erlang.org/doc/man/sets.html#from_list-1
and then:
http://erlang.org/doc/man/sets.html#size-1 instead of length ;)

Reply:
done.